### ISSUE TITLE
Adicionar latitude e longitude às multas

### ISSUE THREAD

**Issue by giggio** [multas/sociais#8]
### História

Enquanto usuário do site ou de uma app
Eu gostaria de informar a latitude e a longitude com o uso de alguma ferramenta
Para que eu possa ter certeza do local exato onde ocorreu a multa
### Critério de aceite

Deve estar disponível no site e na API
O usuário nunca entra Latitude e longitude diretamente, ou ela é derivada de um mapa do Google Maps, ou ela é informada via API, onde as apps podem usar outros recursos, como GPS do celular ou tablet
É opcional
Pode trabalhar em conjunto com a issue de endereços #7

**Comment by brunont** [comment]
Achei aqui q já tem lat e lng no banco... um dia começamos isso...
falta todo o resto, testar cadastro, colocar na API, etc...

**Comment by giggio** [comment]
Ae, já tá em andamento então! :)

**Comment by natanaelmaia** [comment]
Outra opção que talvez seja interessante seria extrair a latitude e longitude diretamente do EXIF da foto...

**Comment by giggio** [comment]
Interessante! Se o cara não informar, a gente informa pra ele que parece que a foto foi tirada em tal lugar, e se ele quer acrescentar!

**Comment by natanaelmaia** [comment]
Exatamente... Isso facilitaria bastante pro usuário :) Vale lembrar que a latitude/longitude do EXIF da foto representa o local exato onde a foto foi tirada, diferente da latitude/longitude informada pelo app no upload, que pode ser realizado num momento posterior já em outra localização. Tenho a impressão que o Instagram trabalha dessa forma...

**Comment by natanaelmaia** [comment]
https://github.com/picuous/exifr
ou
https://github.com/remvee/exifr

**Comment by giggio** [comment]
Acho que se a app informar a lat/lng a que vem no exif deve ser ignorada, porque a da app provavelmente é mais precisa, com alguma correção do usuário. No pior dos casos é a mesma, já que o cara tira a foto e submete em seguida.
